deploy/helm/sumologic/values.yaml

```diff
@@ -2208,17 +2208,18 @@ otelcol:
       # logging:
       #   loglevel: debug
       otlphttp:
-        endpoint: ${SUMO_ENDPOINT_DEFAULT_TRACES_SOURCE}
+        traces_endpoint: ${SUMO_ENDPOINT_DEFAULT_TRACES_SOURCE}
```

Comment:
This is a breaking change for end users. Is  tracing via otelcol considered a beta feature so that it is expected or do we need to explicitly warn users whenever we release next version?

Reply:
@pmaciolek what do you think about it?

Reply:
Why is this considered a breaking change @pmalek-sumo ?

Reply:
Yup. Someone could've specified a custom `endpoint` and now it won't work i.e. it will have to be changed to `traces_endpoint`, otherwise this custom value won't be used when installing the chart.

Reply:
We usually keep this kind of changes as PRs with `breaking change` label.
They are picked up during next breaking change release.